pkgs/desktops/pantheon/desktop/elementary-greeter/default.nix

```diff
@@ -80,7 +70,6 @@ stdenv.mkDerivation rec {
     libhandy
     lightdm
     mutter
-    wingpanel-with-indicators
```

Comment:
The greeter uses the panel as its own process for a while so I removed this from buildInputs but this still needs to stay in PATH.